crates/update/src/cli/list.rs

```diff
@@ -0,0 +1,29 @@
+use spacetimedb_paths::SpacetimePaths;
+
+/// List installed SpacetimeDB versions.
+#[derive(clap::Args)]
+pub(super) struct List {
+    /// List all versions available to download and install.
+    #[arg(long)]
+    all: bool,
+}
+
+impl List {
+    pub(super) fn exec(self, paths: &SpacetimePaths) -> anyhow::Result<()> {
+        let current = paths.cli_bin_dir.current_version()?;
+        let versions = if self.all {
+            let client = super::reqwest_client()?;
+            super::tokio_block_on(super::install::available_releases(&client))??
+        } else {
+            paths.cli_bin_dir.installed_versions().collect::<Result<Vec<_>, _>>()?
+        };
+        for ver in versions {
+            print!("{ver}");
+            if Some(&ver) == current.as_ref() {
+                print!(" (current)");
+            }
+            println!();
+        }
+        Ok(())
+    }
+}
```

Comment:
I get this error:
```
$ cargo run -- version list
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.38s
     Running `target/debug/spacetimedb-cli version list`
Error: No such file or directory (os error 2)
```

Is it just because I'm running from `target` instead of a proper install?

Reply:
Probably, yes. I can improve the error message.